wp-content/mu-plugins/pantheon/pantheon-try-gutenberg-mods.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * Should we proceed with adding the
+ * Try Gutenberg panel mods?
+*/
+$mod_try_gutenberg = apply_filters( 'show_pantheon_try_gutenberg_mods', ( ! defined('DISABLE_PANTHEON_TRY_GUTENBERG_MODS') || ! DISABLE_PANTHEON_TRY_GUTENBERG_MODS ) );
```

Comment:
Having a constant _and_ a filter seems overkill. I think the constant would be sufficient.

Also, and this is a super minor nit, mu plugins are loaded in global scope so `$mod_try_gutenberg` would technically become a global variable. Given the lack of prefix, this variable could collide with some other usage, however unlikely. It's probably best to not introduce a global when possible.